CMakeLists.txt

```diff
@@ -96,12 +104,6 @@ if (NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Release)
 endif ()
 
-# If ccache exists, use it to speed up compiling
-find_program(CCACHE_FOUND ccache)
-if(CCACHE_FOUND)
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
-endif(CCACHE_FOUND)
-
```

Comment:
ccache is not safe enough, especially when switching from debug to release.

Besides, CMake has already supported incremental compile.